running/README.md

```diff
@@ -590,6 +590,49 @@ This can be achieved by setting the `max_jobs` property of a partition in the `s
 
 ```
 
+## Execution modes
+
+Reframe 2.5 permits you to define different *execution modes* of the framework.
```

Comment:
Would it be more precise if we state something like:
Starting from version 2.5, ReFrame ...

What do you think?